src/dmd/dsymbolsem.d

```diff
@@ -1841,25 +1841,122 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             if (!pd.decl)
                 return;
 
+            StringExp verifyMangleString(ref Expression e)
```

Comment:
This is quite a large function, but there's not much of a clue as to what it does and what it returns.

Reply:
https://github.com/dlang/dmd/pull/12283